### PR TITLE
Chore: APP-1223 - Bump SDK version 

### PR DESCRIPTION
**Pull request by Fabricevladimir**
## Description
- Bumps SDK version to 0.12.0-alpha
- Updated DAO factory address

Task: [APP-1223](https://aragonassociation.atlassian.net/browse/APP-1223)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

### CODE REVIEW

File: packages/web-app/src/hooks/useClient.tsx [hunk 1]
```diff
@@ -44,7 +44,7 @@ export const UseClientProvider: React.FC = ({children}) => {
           },
         },
       ],
-      daoFactoryAddress: '0x8B4Ca38524fCeCbD5acA39C7cd2f3B762B1d91Bf', // TODO: remove temporary until SDK updates
+      daoFactoryAddress: '0xd52CA6BbfC11627aF9aC3496CAb975ae20E70A81', // TODO: remove temporary until SDK updates
```
Comment (WissenIstNacht) [inline packages/web-app/src/hooks/useClient.tsx:47]: I lost the link. Where can this one be found again?

**Review by WissenIstNacht — APPROVED** [review]
🚀

File: packages/web-app/src/hooks/useClient.tsx (hunk 1 above)
Comment (Fabricevladimir) [reply packages/web-app/src/hooks/useClient.tsx:47]: https://github.com/aragon/core/blob/develop/active_contracts.json

**Review by sepehr2github — APPROVED** [review]
Thanks, Fabrice

**Pull request merged**